Classes/Command/DatabaseStorageCommandController.php

```diff
@@ -0,0 +1,156 @@
+     * @Flow\Inject
+     */
+    protected DatabaseStorageService $databaseStorageService;
+
+    /**
+     * @Flow\InjectConfiguration(package="Wegmeister.DatabaseStorage", path="cleanup")
+     * @var array|null
+     */
+    protected ?array $storageCleanupConfiguration;
+
+    /**
+     * Deletes entries of configured storages older than configured date interval
+     */
+    public function cleanUpConfiguredStoragesCommand(): void
+    {
+        $this->outputFormatted('<b>Cleanup of configured storages</b>');
+        $this->outputLine('');
+
+        if (empty($this->storageCleanupConfiguration)) {
+            $this->outputFormatted('No cleanup configuration found.', [], 4);
+            $this->outputFormatted('Please configure the cleanup for Wegmeister.DatabaseStorage in Settings.yaml.', [], 4);
+            return;
+        }
+
+        $results = [];
+        foreach ($this->storageCleanupConfiguration as $storageIdentifier => $storageCleanupConfiguration) {
+            $results[$storageIdentifier] = ['storageIdentifier' => $storageIdentifier, 'messages' => ''];
+
+            // Check if the date interval is valid
+            try {
+                $newDateInterval = $this->getDateIntervalFromConfiguration($storageIdentifier);
+            } catch (\Exception $exception) {
+                $results[$storageIdentifier]['messages'] .= $exception->getMessage() . PHP_EOL;
+                continue;
+            }
+
+            // Check if we have entries for the storage identifier
+            $daysToKeepData = $this->getDaysToKeepFromConfiguredInterval($newDateInterval);
+            $amountOfEntries = $this->databaseStorageService->getAmountOfEntriesByStorageIdentifier($storageIdentifier);
+            if ($amountOfEntries === 0) {
+                $results[$storageIdentifier]['messages'] .= sprintf('No entries found in storage "%s".', $storageIdentifier) . PHP_EOL;
+                continue;
+            }
+
+            // Cleanup the storage
+            $results[$storageIdentifier]['messages'] .= vsprintf('Removing entries from storage "%s" older than %s days...', [$storageIdentifier, $daysToKeepData]) . PHP_EOL;
+            $amountOfOutdatedEntries = $this->databaseStorageService->cleanupByStorageIdentifierAndDateInterval($storageIdentifier, $newDateInterval);
+            $results[$storageIdentifier]['messages'] .= vsprintf('Removed %s entries from storage "%s" (%s entries in total).', [$amountOfOutdatedEntries, $storageIdentifier, $amountOfEntries]);
+        }
+
+        $this->output->outputTable($results, ['storageIdentifier', 'messages'], 'Cleanup results');
+    }
+
+    /**
+     * Deletes entries of all storages older than given date interval.
+     * You can also skip the configured storages.
+     *
+     * @param string $dateInterval
+     * @param bool $skipConfiguredStorages
+     */
+    public function cleanupAllStoragesCommand(string $dateInterval, $skipConfiguredStorages = true): void
```

Comment:
Is there a way to set a boolean cli argument to `false` in Flow? If not, the parameter should be sth. like `$includeConfiguredStorages = false` or even `$includeAll...`